modules/processing.py

```diff
@@ -321,7 +323,7 @@ def create_random_tensors(shape, seeds, subseeds=None, subseed_strength=0.0, see
         # the way I see it, it's better to do this on CPU, so that everyone gets same result;
         # but the original script had it like this, so I do not dare change it for now because
         # it will break everyone's seeds.
-        noise = devices.randn(seed, noise_shape)
+        noise = noise_override if noise_override != None else devices.randn(seed, noise_shape)

```

Comment:
should this be `is not None` ?

Reply:
I can change it to that if you prefer, but it's really a semantic difference. `!=` checks for value inequality, `is not` checks for reference inequality. Since all values of `None` in python are technically a reference to the same object, they're functionally equivalent here.

Reply:
It's no big deal. A PEP8 linter will raise a warning: https://peps.python.org/pep-0008/#programming-recommendations

Reply:
fair enough, I'll change it then.